go.mod

```diff
@@ -27,7 +26,7 @@ require (
 	golang.org/x/net v0.18.0
 	golang.org/x/sync v0.3.0
 	google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1
-	google.golang.org/grpc v1.55.0
+	google.golang.org/grpc v1.56.3
```

Comment:
There are higher versions but I guess this minimizes the delta.

Reply:
> There are higher versions but I guess this minimizes the delta.

exactly why I picked it. It's one of the versions from here https://security.snyk.io/vuln/SNYK-GOLANG-GOOGLEGOLANGORGGRPC-5953328